core/src/main/scala/com.snowplowanalytics.snowplow.collector.core/Service.scala

```diff
@@ -139,11 +140,13 @@ class Service[F[_]: Sync](
   }
 
   override def preflightResponse(req: Request[F]): F[Response[F]] = Sync[F].pure {
+    val allowedHeaders = (List("Content-Type", "SP-Anonymous") ++ config.cors.allowedCorsHeaders).map(CIString(_))
```

Comment:
What about just making `Content-Type` and `SP-Anonymous` a part of the `allowedCorsHeaders` setting? That would avoid rebuilding this list on every request. And actually `allowedCorsHeaders` is a misnomer if it does not include _all_ allowed headers.

Reply:
Do you mean as part of the config? My only concern would be if people remove this from a default config.